### DISCUSSION TITLE
Welcome to wpfs2023-qwik-city Discussions!

### DISCUSSION THREAD

**Discussion by ludizhan** [ludizhan/wpfs2023-qwik-city#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tbondwilkinson** [comment]
COOL I'M SO EXCITED

**Reply by tbondwilkinson** [reply]
Here's a reply on the first comment

**Comment by tbondwilkinson** [comment]
Here's another comment!
